src/v2/options.ts

```diff
@@ -350,6 +365,13 @@ export function optionsToEndpoint(
     }
     return region;
   });
+  convertIfPresent(
+    endpoint,
+    opts,
+    'secretEnvironmentVariables',
+    'secrets',
+    (secrets) => secrets.map((secret) => ({ secret, key: secret }))
```

Comment:
I'm confused. Isn't this specifying both the env var and the secret name? DIdn't we not want to do that in the SDK but in the CLI since (in the future) the build process needs to not set the secret name so that extensions can?

Reply:
ah excellent catch.